app/controllers/dashboard/schools_controller.rb

```diff
@@ -18,6 +17,7 @@ def index
 
     def edit
       authorize! :edit, School
+      @available_weeks = Week.selectable_on_school_year
```

Comment:
school edit ; show all weeks on current school year otherwise we lose the info. leading to sentry#1345155452